src/AuthModule/Components/Register/Register.tsx

```diff
@@ -1,11 +1,347 @@
+    }
+    setIsLoading(false);
+  }
+
+  function appendToFormData(values: any) {
+    const registerForm = new FormData();
+    registerForm.append("userName", values.userName);
+    registerForm.append("email", values.email);
+    registerForm.append("country", values.country);
+    registerForm.append("phoneNumber", values.phoneNumber);
+    registerForm.append("password", values.password);
+    registerForm.append("confirmPassword", values.confirmPassword);
+    registerForm.append("profileImage", values.profileImage[0]);
+    return registerForm;
+  }
+
+  
+ 
+  const onSubmit = async (values: any) => {
+    localStorage.setItem("register-Email", values?.email);
+
+    setIsLoading(true);
+    const formData = appendToFormData(values);
+    handleRegister(formData);
+  };
+
+  const togglePasswordVisibility = () => {
+    setShowPassword(!showPassword);
+  };
+
+  const toggleConfirmPasswordVisibility = () => {
+    setShowConfirmPassword(!showConfirmPassword);
+  };
+
+
+
+function handlePic(e:any) {
+  setUserImage(URL.createObjectURL(e.target.files[0]))
+  
+  
+}
   return (
     <>
-    <div className='Register vh-100 overflow-auto pageOverflow'>
-      Register
+      <div className="auth-container py-5 p-md-0 d-flex flex-column justify-content-center align-items-center">
+        <div className="logo">
+          <img className="form-logo pb-2" src={logo} alt="logo" />
+        </div>
+        <form   
+                  onSubmit={handleSubmit(onSubmit)}
+          className="form col-md-8 col-sm-9 col-10 bg-info rounded-4 p-md-5  p-3"
+        >
+         <div className=" pt-3">
+           <p className=" text-light mb-0">welcome to PMS</p>
+           <h2 className=" form-Name mb-3">
+             <span>C</span>reate New Account
+           </h2>
+           </div>
+          
+          <div className="upload mx-auto my-3 d-flex justify-content-center align-items-center ">
+            <img className="  m-0 p-0" src={userImage||defaultImage} alt="selectImage" />
+
+            <div className="round  text-center  d-flex align-items-end  text-center justify-content-center mx-auto  ">
+              <div className="h4 carousel mx-auto text-center position-relative start-0 w-100">
+                <input
+                onChangeCapture={(e)=>handlePic(e)}
+                  {...register("profileImage")}
+                  className="m-0 p-0 position-relative"
+                  type="file"
+                />
+                <i className="fa-solid fa-camera"></i>
+              </div>
+            </div>
+          </div>
+
+          <div className="d-flex flex-wrap align-items-center    mt-2 ">
+            <div className=" one-input-group   col-md-6 pe-md-3   my-4  col-12  ">
+              <div
+                className="form-outline my-2  text-start   "
+                data-mdb-input-init
+              >
+                <label className="form-label  fw-medium pb-0 mb-0">User Name</label>
```

Comment:
We can add `htmlFor` attribute to the label and `id` for the input to connect them with each other. This will allow user to focus on the input when clicking on the label.